src/NewsletterEpic/index.tsx

```diff
@@ -151,39 +157,49 @@ const ctaStyles = {
         color: ${palette.neutral[0]};
     `,
 };
+
+type SignUpButtonProps = { onSignUpClick: () => void };
+
+const SignUpButton: React.FC<SignUpButtonProps> = (props: SignUpButtonProps) => {
+    return (
+        <ThemeProvider theme={buttonBrandAlt}>
+            <Button css={ctaStyles.button} onClick={props.onSignUpClick}>
+                Sign up
+            </Button>
+        </ThemeProvider>
+    );
+};
+
 const CTA: React.FC<CTAProps> = (props: CTAProps) => {
     const { subscribeToNewsletter, newsletterId } = props;
 
     const [subscribeClickStatus, setSubscribeClickStatus] =
         useState<SubscribeClickStatus>('DEFAULT');
 
+    const onSignUpClick = () => {
+        setSubscribeClickStatus('IN_PROGRESS');
+
+        subscribeToNewsletter(newsletterId as string)
+            .then(() => setSubscribeClickStatus('SUCCESS'))
+            .catch(() => {
+                setSubscribeClickStatus('FAILURE');
+            });
+    };
+
     switch (subscribeClickStatus) {
         case 'DEFAULT':
+            return <SignUpButton onSignUpClick={onSignUpClick}></SignUpButton>;
```

Comment:
Purely subjective but what do you think about writing this using a self-closing tag? I tend to do that when the component has no children.

```suggestion
            return <SignUpButton onSignUpClick={onSignUpClick} />;
```